mongodb_exporter.go

```diff
@@ -111,7 +117,33 @@ func main() {
 		SocketTimeout:            *socketTimeoutF,
 		SyncTimeout:              *syncTimeoutF,
 	})
-	prometheus.MustRegister(mongodbCollector)
+	prometheus.MustRegister(programCollector, mongodbCollector)
 
 	exporter_shared.RunServer("MongoDB", *listenAddressF, *metricsPathF, promhttp.ContinueOnError)
 }
+
+// initVersionInfo sets version info
+// If binary was build for PMM with environment variable PMM_RELEASE_VERSION
+// `--version` will be displayed in PMM format. Also `PMM Version` will be connected
+// to application version and will be printed in all logs.
+func initVersionInfo() {
+	version.Version = pmmVersion.Version
+	version.Revision = pmmVersion.FullCommit
+	version.Branch = pmmVersion.Branch
+
+	if buildDate, err := strconv.ParseInt(pmmVersion.Timestamp, 10, 64); err != nil {
+		version.BuildDate = time.Unix(0, 0).Format(versionDataFormat)
+	} else {
+		version.BuildDate = time.Unix(buildDate, 0).Format(versionDataFormat)
+	}
+
+	if pmmVersion.PMMVersion != "" {
+		version.Version += "-pmm-" + pmmVersion.PMMVersion
+		kingpin.Version(pmmVersion.FullInfo())
+	} else {
+		kingpin.Version(version.Print(program))
+	}
```

Comment:
Honestly, I'm not sure we need that. `pmmVersion.ShortInfo()` will not mention PMM at all if needed. Maybe we could drop two ways to set a version and to keep only one (ours)?

Reply:
It's about community and internal build (https://github.com/percona/mongodb_exporter/pull/132#issuecomment-467411483). I thought it will be right if we save original info in community version.

Reply:
I dunno. Ok, let's keep it.